### PR TITLE
Stucy opisanie

### PR DESCRIPTION
**Pull request by stucy**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  width:100%;
+  max-height: 300px;
+}
+
+.carousel-inner img {
+  height: 300px;
+  width: 100%
+}
+*/
+#carousel {
```
Comment (marryy) [inline css/main.css]: apply styling to class, not #id

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+h1 {
+  text-align: center;
+}
+.opisanie {
+  background-color: purple;
+  border: solid black 2px;
+  border-radius: 50px;
+}
+/*.carousel-inner{
```
Comment (marryy) [inline css/main.css]: remove commented code

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  max-height: 300px;
+}
+
+.carousel-inner img {
+  height: 300px;
+  width: 100%
+}
+*/
+#carousel {
+  margin: 50px 20px 0px 20px;
```
Comment (marryy) [inline css/main.css]: use shorter syntax (50px 20px 0px)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  </div>
+    <a class="carousel-control-next" href="#carousel" role="button" data-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+
+  </div>
+
+</div>
+<div id="footer">
```
Comment (marryy) [inline index.html]: remove additional <div>

File: index.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    <footer class="footer bg-dark text-white pt-2 mt-2">
+
+        <div class="container text-center text-md-left">
```
Comment (marryy) [inline index.html]: remove < div >. apply more styling to the < footer >

File: index.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+            <div class="row text-center text-md-left mt-2 pb-2">
+
+                <div class="col-md-9 col-lg-9 mx-auto mt-2">
+                    <h6 class="text-uppercase mb-4 font-weight-bold">Company name</h6>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                </div>
+
+                <hr class="w-100 clearfix d-md-none">
```
Comment (marryy) [inline index.html]: remove < hr > . apply more styling to other elements

File: less/main.less [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    left: 0;
+    padding-top: 330px;
+    text-align: center;
+  }
+
+h1 {
+  text-align: center;
+}
+
+.opisanie{
```
Comment (marryy) [inline less/main.less]: rename to 'description' for example

File: less/main.less [hunk 8]
```diff
@@ -0,0 +1,64 @@
+
```
Comment (marryy) [inline less/main.less]: split less file to separate files - for each page, e.g. 'homepage.less', 'about.less', 'contacts.less', etc.

File: less/main.less [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+h1 {
+  text-align: center;
+}
+
+.opisanie{
+ background-color: @secondary-color;
+ border:solid black 2px;
+ border-radius:50px;
```
Comment (marryy) [inline less/main.less]: add space after :  - apply everywhere

**Comment by stucy** [comment]
Made the fixes and merged with develop and it said it is already up to date

**Comment by stucy** [comment]
Fixed everything

**Pull request closed without merging**